resident-default.properties

```diff
@@ -727,6 +729,7 @@ resident.template.summary.success.REVOKE_VID=gen-or-revoke-vid-success-summary
 resident.template.summary.success.GET_MY_ID=get-my-uin-card-success-summary
 resident.template.summary.success.VALIDATE_OTP=verify-my-phone-email-success-summary
 resident.template.summary.success.VID_CARD_DOWNLOAD=vid-card-download-positive-summary
+resident.template.summary.success.AUTHENTICATION_REQUEST=mosip.event.type.AUTHENTICATION_REQUEST
```

Comment:
why do you have the same value for summary, failure and success ?

Reply:
Corrected template will add PR for mosip-data also.

Reply:
https://github.com/mosip/mosip-data/pull/315